docs/member/issuer-profile.md

```diff
@@ -0,0 +1,78 @@
+# Issuer Profile
+
+## 1. What is an Issuer?
+
+A legal entity that manages, registers, and sells securities to support its operations financially is an Issuer. There can be many types of Issuers - Corporations, Domestic or Foreign Governments and Investment trusts. Issuers develop securities such as Bonds, Debentures, G-Secs & NCDS. Issuers are legally responsible for reporting financial conditions, material developments, and other operational activities required by the jurisdiction's regulations.
+
+The AlgoBulls Issuer Profile gives you a detailed view of the issuer’s name, bonds available, transactions, average yield and more. Check the Issuer profile here.
```

Comment:
`Check the Issuer profile here.`

`here` was supposed to be a link here right?

Reply:
Link added